diagnostic.md

```diff
@@ -33,7 +40,7 @@ y = 10;
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
 <!-- solution below -->
-
```

Comment:
You left out the first bit of the question. The variables `values` may change but the declaration of the variable does not.